lib/screens/map_view.dart

```diff
@@ -136,23 +136,38 @@ class _MapViewScreenState extends State<MapViewScreen> {
 
   @override
   Widget build(BuildContext context) {
+    final navigationArgs = (ModalRoute.of(context)!.settings.arguments);
+    final collectionId =
+        (navigationArgs as Map<String, String>)['collectionId'];
```

Comment:
I keep getting the below error when logging in with google on the last tutorial page or answering a math question correctly for the existing account. 
"_CastError (type 'Null' is not a subtype of type 'Map<String, String>' in type cast)"